src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -15,6 +15,7 @@
 import java.io.*;
```

Comment:
Just noticed this, but the style guide says no wildcard import.

Reply:
Changed to import java.io.IOException in both UserServlet and ForumServlet